sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -248,7 +249,7 @@ private[hive] object HadoopTableReader extends HiveInspectors {
    * instantiate a HadoopRDD.
    */
   def initializeLocalJobConfFunc(path: String, tableDesc: TableDesc)(jobConf: JobConf) {
-    FileInputFormat.setInputPaths(jobConf, path)
+    jobConf.set("mapred.input.dir", StringUtils.escapeString(path.toString()))
```

Comment:
Instead of setting the conf using the key, can we still use `FileInputFormat.setInputPaths`? Like
```
FileInputFormat.setInputPaths(jobConf, StringUtils.escapeString(path))
```